labrad/fasttypes.c

```diff
@@ -0,0 +1,2364 @@
+#include "Python.h"
+
+/*
+NOTES:
+- Python.h modifies assert() so it does not trigger program termination on failure. exit() terminates the interpreter.
+- small memory allocations assumed to work.
+- ndarrays not allowed to contain objects with structure.
+
+Common brief variable names and meanings
+i, j : for-loop counters
+len, n : the length of a list/tuple/array/string/etc object
+lobj : list Python object
+obj : generic Python object
+robj : the Python object that will be returned by a function
+sobj : string Python object
+tobj : temporary Python object
+*/
+
+#define TRUE 1
+#define FALSE 0
+#define SUCCESS 1
+#define EXCEPTION_RAISED -1
+#define BIG_ENDIAN ">"
+#define LITTLE_ENDIAN "<"
```

Comment:
On my system, BIG_ENDIAN and LITTLE_ENDIAN are already preprocessor macros.  I will rename.